.changeset/entity-descriptions.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/memory': patch
+---
+
+Improve memory curation instructions.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)